packages/fx-core/src/common/samples.ts

```diff
@@ -71,8 +70,8 @@ class SampleProvider {
       // prerelease build version always use branch head for prerelease.
       this.branchOrTag = SampleConfigBranchForPrerelease;
     } else if (version.includes("rc")) {
-      // rc version(before next stable TTK) always use prerelease tag
-      this.branchOrTag = SampleConfigTagForRc;
+      // if there is a breaking change, the tag is not used by any stable version.
```

Comment:
Do we ship v2.4.0 in rc build?

Reply:
Yes, if I understand correctly, rc is the version that we will test before stable release. It's not published to marketplace.